### ISSUE TITLE
Screen reader does not announce complete information over dates while navigating using Alt+ctrl+arrow keys with NVDA

### ISSUE THREAD

**Issue by huangzumeihan** [microsoft/fluentui#20423]
Test Environment:
URL: https://developer.microsoft.com/en-us/fluentui#/controls/web/datepicker

Repro Steps:
1. Open the above URL 
2. Navigate to Default DatePicker tab and activate it.
3.Enable NVDA & Navigate over Calendar button activate it >Custom date range>Start date calendar .
4. Observe the issue as Screen reader does not announce complete information over dates while navigating using Alt+ctrl+arrow keys with NVDA

Actual:
Screen reader does not announce complete information over dates while navigating using Alt+ctrl+arrow keys with NVDA.
NVDA announce -row 2 Sa column 1.

Expected:
Screen reader does not announce complete information over dates while navigating using Alt+ctrl+arrow keys with NVDA.
NVDA should announce as -row 2  Sa  column 7  button  2
![image](https://user-images.githubusercontent.com/31198399/139629145-6d5529e1-bbcd-49a6-8432-2ee3bf1c137c.png)
